dev/benchmarks/macrobenchmarks/pubspec.yaml

```diff
@@ -23,6 +23,11 @@ dependencies:
       url: https://github.com/flutter/gallery.git
       ref: d8b4858f88ab9006c15a5c9e197386623ea2e12c
 
+  web_benchmarks:
+    git:
+      url: https://github.com/pennzht/web_benchmarks.git
+      ref: 40a483181861e6cd84dd14a6d31cb45a966f8d59
```

Comment:
In order to fix the issue in question this will become a normal pub dependency, correct?

Reply:
Yes, this will become something like `web_benchmarks: <version>` similar to `animations: 1.1.2` below, once the pub package is published.

Reply:
Perfect!